pages/_document.tsx

```diff
@@ -0,0 +1,11 @@
+import { Head, Html, Main, NextScript } from "next/document";
+
+export default function Document() {
+  return (
+    <Html data-theme="evie">
```

Comment:
Data theme evie? You wanna sell me something here?

Reply:
it's the evie color scheme for daisyui.